content/post/culture_of_partnership.md

```diff
@@ -0,0 +1,74 @@
+# A Culture of Partnership
+
+During my time leading an analytics and data science team, I spent a lot of time thinking about how an ideal analytics team should operate -- how should the team work together, how should the team prioritize their work, and how can the team most effectively partner with the broader organization to generate business value.
```

Comment:
nit - I think 
```
how should the team work together, how should the team prioritize their work, and how can the team most effectively partner
```
should read
```
how the team should work together, how the team should prioritize their work, and how the team can most effectively partner
```

otherwise it should end with a question mark, right?